### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by yahooo-m** [hkchengrex/MMAudio#58]
Hi， nice work!
I want to reproduce this work on my device, which is A100 40G GPUs. Do you think it will be realized?

**Comment by hkchengrex** [comment]
Hi, thank you for your interest.
Both the small/medium models should work easily on eight A100-40Gs. You might need to implement gradient checkpointing/reduce batch sizes to train the large model.
